universal-application-tool-0.0.1/universal-application-tool-0.0.1.iml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean to check this change in?

Reply:
No

Reply:
No, I think that was added by IntelliJ

Reply:
No, it must have been put in by IntelliJ

Reply:
Would you remove it from the PR by running `git rm path/to/file`? You could also add it to `.gitignore` if you will.

Reply:
Done by using git rm

Reply:
Nice! Thanks.